### PR TITLE
Edit Asset

### PR DESCRIPTION
**Pull request by vivianliu0**
https://www.loom.com/share/46ea88caa61c4a469d5599e95ad02516

### CODE REVIEW

File: src/Components/Admin/AssetModelViewer/assetModelViewer.js [hunk 1]
```diff
@@ -149,6 +162,8 @@ export default function AssetModelViewer({
                         value={name}
                         onChange={handleNameChange}
                         variant="outlined"
+                        error={!empty(nameError)}
```
Comment (aaronabraham311) [inline src/Components/Admin/AssetModelViewer/assetModelViewer.js:165]: Oh interesting why do we have `empty`? Couldn't we just compare against `""`? Or am I missing something?

**Review by aaronabraham311 — COMMENTED** [review]
Think this is pretty good. @ahmedhamodi thoughts on the error message?

File: src/Components/Admin/AssetModelViewer/assetModelViewer.js (hunk 1 above)
Comment (vivianliu0) [reply src/Components/Admin/AssetModelViewer/assetModelViewer.js:165]: Oh yup, we can do that instead. I just copied the validation logic from login.js which was using this for a slightly more complicated use case.

Comment (ahmedhamodi) [reply src/Components/Admin/AssetModelViewer/assetModelViewer.js:165]: We could, I think empty() is safer and also checks for null or other "empty"-type states. 

This covers the other states they detect.
https://www.npmjs.com/package/is-empty

**Review by ahmedhamodi — COMMENTED** [review]
image

I'm able to fully remove the name and save. We should not allow this.

Note: actual save results in error, so definitely should not be allowing this:
image

**Comment by ahmedhamodi** [comment]
> thoughts on the error message?

I think it's fine!

**Comment by vivianliu0** [comment]
> I'm able to fully remove the name and save. We should not allow this.

Hmm weird, on my end I'm seeing that it disallows an empty field
image

**Comment by ahmedhamodi** [comment]
> Hmm weird, on my end I'm seeing that it disallows an empty field

Oh I was on an older version of your branch for some reason o_o, I pulled and yeah I see that behaviour.

Okay, this lgtm then!

**Review by ahmedhamodi — APPROVED** [review]

File: src/Components/Admin/AssetModelViewer/assetModelViewer.js [hunk 2]
```diff
@@ -15,6 +15,7 @@ import MenuItem from "@material-ui/core/MenuItem";
 import FormControl from "@material-ui/core/FormControl";
 import InputLabel from "@material-ui/core/InputLabel";
 import { ObjectTypes } from "./objectTypes.ts";
+import empty from "is-empty";
```
Comment (ahmedhamodi) [inline src/Components/Admin/AssetModelViewer/assetModelViewer.js:18]: You'll need to remove the unused import now too with the usage removed.

**Review by ahmedhamodi — APPROVED** [review]

**Pull request merged**